niworkflows/interfaces/nibabel.py

```diff
@@ -125,7 +125,8 @@ def _run_interface(self, runtime):
         self._results["out_files"] = []
         for i, img_3d in enumerate(nb.four_to_three(img)):
             out_file = str(
-                Path(fname_presuffix(in_file, suffix=f"_idx-{i:03}")).absolute()
+                Path(fname_presuffix(in_file, suffix=f"_idx-{i:03}",
+                                     newpath=runtime.cwd)).absolute()
```

Comment:
runtime.cwd should be sufficient i believe?
```suggestion
                                     newpath=runtime.cwd))
```

Reply:
Removed `Path` altogether